packages/eth-edge/src/controllers/wallets.controller.ts

```diff
@@ -22,4 +25,11 @@ export class WalletsController {
         const getWalletReq = await useValidate(getWalletsSchema, params);
         return this.walletsService.findAll(getWalletReq.id);
     }
+
+    @Delete("/users/:UserId/wallets/:address")
+    async delete(@Body() body: DeleteWalletRequest): Promise<Wallet> {
+        const deleteWalletReq = await useValidate(deleteWalletSchema, body);
+        console.log(deleteWalletReq);
+        return await this.walletsService.delete(deleteWalletReq);
```

Comment:
No need to await a return statement

```suggestion
        return this.walletsService.delete(deleteWalletReq);
```

Reply:
fix this